### ISSUE TITLE
Concept has no reference to namespace or actual full property name, should it?

### ISSUE THREAD

**Issue by cwd-mparsons** [CredentialEngine/CompetencyFrameworks#91]
@stuartasutton 
I have published a few concept schemes to the sandbox and am curious. 
https://sandbox.credentialengineregistry.org/graph/ce-bbb6954d-180d-47f2-82d1-a942c97a1a71

![image](https://user-images.githubusercontent.com/26332112/54450522-953e7480-471e-11e9-9d51-5cce6b9964d8.png)

The latter is for Current Military of audience. There is no indication of the actual property name of audience:CurrentMilitary, or the actual namespace. 
The inScheme points to https://credreg.net/ctdl/terms/Audience/json. Can someone consistently use the latter to format namespace. That is, is the latter the common protocol?
I don't recall if we already had one exception to this pattern.

I am not sure if this is an issue or if the steps to resolve would be clear to developers.

**Comment by siuc-nate** [comment]
In the original data ( https://credreg.net/ctdl/terms/Audience/json ), the `skos:inScheme` for `audience:CurrentMilitary` is `ceterms:Audience`.  

When you resolve `ceterms` part via the `@context`, you get `http://purl.org/ctdl/terms/`, which should then be concatenated (per the way JSON-LD works) with the rest of `ceterms:Audience`, (that is to say, with `Audience`) yielding `http://purl.org/ctdl/terms/Audience`, which is the correct URI (and will redirect via PURL to https://credreg.net/ctdl/terms/Audience ).

In other words, in the published data, skos:inScheme _could_ be http://purl.org/ctdl/terms/Audience but since the published data also includes the same `@context` URI, then you could also publish with just `"skos:inScheme": "ceterms:Audience"` - though it should probably be multi-value.

**Comment by Lomilar** [comment]
I agree that both are valid. I am of the opinion that Values should be fully qualified URLs where possible. Keys are not held to such stringent requirements.

**Comment by siuc-nate** [comment]
The inScheme URI shouldn't have `/json` on the end of it.

**Comment by woodkri** [comment]
Current behavior for CTDL export of concept schemes is a bit different from the screenshot above. The inScheme URI is now the framework's registry URL. We added a link back to the canonical URI of the concept in skos:exactMatch, so you can see the Audience namespace there.

![CurrentMilitary](https://user-images.githubusercontent.com/33300923/61004951-2b815080-a335-11e9-85bf-a83d5c9ad2e3.PNG)

**Comment by Lomilar** [comment]
Is this still a concern?

**Comment by stuartasutton** [comment]
I am with Fritz (https://github.com/CredentialEngine/CompetencyFrameworks/issues/91#issuecomment-510193443) "that Values should be fully qualified URLs".

**Comment by cwd-mparsons** [comment]
I understand the importance of the URL. My question was on clarity. 
Example:
- Concept of: Advanced Level
- skos:inScheme: "https://credentialengineregistry.org/resources/ce-8943ae15-ff3a-f989-623c-b4fa43e34aa6",
- Name from the latter is Audience Level
- It has a source value of 
 "ceasn:source": "https://credreg.net/ctdl/terms/AudienceLevel"
So the fully qualified property is **audienceLevel: AdvancedLevel**.

What about the Beta frameworks?
Level 1 has 
- skos:inScheme: "https://credentialengineregistry.org/resources/ce-75a38ea0-319d-4748-8bb0-a6dc4c68feb1"
- the name of the conceptScheme is Connecting Credentials: A Beta Credentials Framework - Levels
- the latter has a source of a PDF
- what is the namespace or fully qualified property for Level1 etc?

It may not matter, if there is no need to express or display level1 like betaframe:level1.

**Comment by Lomilar** [comment]
The "best practices" daemon on my shoulder says: "Understanding of something referred to by a URL should, as little as possible, be encoded within that URL. Its a locator, not a substitute for the data."

Obviously, URL based vocabularies violate this horribly, and this is also super annoying, having to resolve URLs all the time just to figure out what that URL means.
